dask_cuda/local_cuda_cluster.py

```diff
@@ -39,96 +39,95 @@ def __init__(self, *args, **kwargs):
+    This assigns a different ``CUDA_VISIBLE_DEVICES`` environment variable to each Dask
     worker process.
 
-    For machines with a complex architecture mapping CPUs, GPUs, and network
-    hardware, such as NVIDIA DGX-1 and DGX-2, this class creates a local
-    cluster that tries to respect this hardware as much as possible.
+    For machines with a complex architecture mapping CPUs, GPUs, and network hardware,
+    such as NVIDIA DGX-1 and DGX-2, this class creates a local cluster that tries to
+    respect this hardware as much as possible.
 
-    It creates one Dask worker process per GPU, and assigns each worker process
-    the correct CPU cores and Network interface cards to maximize performance.
-    If UCX and UCX-Py are also available, it's possible to use InfiniBand and
-    NVLink connections for optimal data transfer performance.
+    Each worker process is automatically assigned the correct CPU cores and network
+    interface cards to maximize performance. If UCX and UCX-Py are available, InfiniBand
+    and NVLink connections can be used to optimize data transfer performance.
 
     Parameters
     ----------
-    CUDA_VISIBLE_DEVICES: str or list
-        String or list ``"0,1,2,3"`` or ``[0, 1, 2, 3]`` to restrict activity to
-        different GPUs.
-    device_memory_limit: int, float or str
-        Specifies the size of the CUDA device LRU cache, which is used to
-        determine when the worker starts spilling to host memory.  This can be
-        a float (fraction of total device memory), an integer (bytes), a string
-        (like ``"5GB"`` or ``"5000M"``), or ``"auto"``, ``0``, or ``None`` to disable
-        spilling to host (i.e., allow full device memory usage). Default is ``0.8``,
-        80% of the worker's total device memory.
-    interface: str
-        The external interface used to connect to the scheduler, usually
-        an ethernet interface is used for connection, and not an InfiniBand
-        interface (if one is available).
-    threads_per_worker: int
-        Number of threads to be used for each CUDA worker process.
-    protocol: str
-        Protocol to use for communication, e.g., ``"tcp"`` or ``"ucx"``.
-    enable_tcp_over_ucx: bool
-        Set environment variables to enable TCP over UCX, even if InfiniBand
-        and NVLink are not supported or disabled.
-    enable_infiniband: bool
-        Set environment variables to enable UCX InfiniBand support, requires
+    CUDA_VISIBLE_DEVICES : str or list of int, optional
+        String (like ``"0,1,2,3"``) or  list (like ``[0, 1, 2, 3]``) to restrict
+        activity to different GPUs.
+    n_workers : int, optional
+        Number of workers. Will override the value of ``CUDA_VISIBLE_DEVICES`` if
+        specified.
+    threads_per_worker : int, default 1
+        Number of threads to be used for each Dask worker process.
+    processes : bool, default True
+        TODO
```

Comment:
Is this parameter required to extend a `LocalCluster`? Looking through the code, all it really does is raise a `ValueError` if it's set false, and it isn't used when calling the super constructor:

https://github.com/rapidsai/dask-cuda/blob/7a6be90e1a72d1caec7f77ce1221d8d6c31134c4/dask_cuda/local_cuda_cluster.py#L286-L290

Reply:
I don't think we need it, it's probably something that there was an idea in mind at the early days of dask-cuda, but it's not necessary at the moment so I'd just remove it.

Reply:
Opened up a separate PR addressing that - #562.